### PR TITLE
Cleaning up the docs a bit

### PR DESCRIPTION
**Pull request by iwoplaza**
- Skipping type-checking of the tutorial code snippets (for now).
- Showing draft posts only in DEV.

### CODE REVIEW

File: apps/typegpu-docs/astro.config.mjs [hunk 1]
```diff
@@ -76,13 +84,22 @@ export default defineConfig({
             //   label: 'Parametrized Functions',
             //   slug: 'guides/parametrized-functions',
             // },
+          ]),
+        },
+        DEV && {
+          label: 'Tutorials',
+          items: [
+            {
+              label: 'Making Triangles Fly',
```
Comment (iwoplaza) [inline apps/typegpu-docs/astro.config.mjs]: A proposal for an alternate name for the tutorial 👀. Thoughts?

Comment (mhawryluk) [reply apps/typegpu-docs/astro.config.mjs]: it's cool, though I personally prefer the old one. and I think the name in the menu should match the guide's heading

File: apps/typegpu-docs/astro.config.mjs [hunk 2]
```diff
@@ -40,7 +44,7 @@ export default defineConfig({
       social: {
         github: 'https://github.com/software-mansion/TypeGPU',
       },
-      sidebar: [
+      sidebar: stripFalsy([
```
Comment (mhawryluk) [inline apps/typegpu-docs/astro.config.mjs:47]: maybe we can bring back the typed-wgsl doc in the dev version?

File: apps/typegpu-docs/astro.config.mjs (hunk 1 above)
Comment (reczkok) [reply apps/typegpu-docs/astro.config.mjs]: Maybe we could merge them? 
'Triangle to boids - making triangles fly'

File: apps/typegpu-docs/src/content/docs/tutorials/triangle-to-boids/index.mdx [hunk 3]
```diff
@@ -19,98 +19,114 @@ In this tutorial, we will explore the capabilities of TypeGPU by transforming a
...
+  <TabItem label="triangle.ts">
+  ```ts
+  import tgpu from 'typegpu';
 
-<Aside type="tip">
+  // Create the root
+  const root = await tgpu.init();
 
-We're creating a canvas by running the `addElement` function from our example toolkit, but that is not the only way! You can use the standard `document.createElement('canvas')` and append it to the document.
+  // Accessing the HTML canvas
```
Comment (mhawryluk) [inline apps/typegpu-docs/src/content/docs/tutorials/triangle-to-boids/index.mdx]:
```suggestion
  // Access the HTML canvas
```

File: apps/typegpu-docs/src/content/docs/tutorials/triangle-to-boids/index.mdx [hunk 4]
```diff
@@ -19,98 +19,114 @@ In this tutorial, we will explore the capabilities of TypeGPU by transforming a
...
+  const canvas = document.getElementById('canvas');
+  const context = canvas.getContext('webgpu') as GPUCanvasContext;
 
-</Aside>
+  // configure the canvas context
```
Comment (mhawryluk) [inline apps/typegpu-docs/src/content/docs/tutorials/triangle-to-boids/index.mdx]:
```suggestion
  // Configure the canvas context
```

**Review by mhawryluk — APPROVED** [review]
LGTM 👍

**Review by reczkok — APPROVED** [review]
Great work 👷‍♂️🛠️

File: apps/typegpu-docs/astro.config.mjs (hunk 1 above)
Comment (iwoplaza) [reply apps/typegpu-docs/astro.config.mjs]: image

**Pull request merged**